aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -819,6 +820,24 @@ def __init__(
         """Amazon API Gateway REST and HTTP API v1 payload resolver"""
         super().__init__(ProxyEventType.APIGatewayProxyEvent, cors, debug, serializer, strip_prefixes)
 
+    # override route to ignore trailing "/" in routes for REST API
+    def route(
+        self,
+        rule: str,
+        method: Union[str, Union[List[str], Tuple[str]]],
+        cors: Optional[bool] = None,
+        compress: bool = False,
+        cache_control: Optional[str] = None,
+    ):
+        # remove trailing "/" character from route rule for correct routing behaviour
```

Comment:
```suggestion
        # NOTE: see #1552 for more context.
```

Adding `NOTE` to ease highlighting when reading code. Also a brief mention about the issue for more context, as we already have a comment about trailing on LOC 823.